round.py

```diff
@@ -200,9 +224,48 @@ async def challenge(self, ctx, *users: discord.Member):
                 await ctx.send(f"{i.name} is already in a round!")
                 return
 
+        check = False 
+        start_time = get_time()
+        First = True
+        while get_time() - start_time < 60:
+            if First:
+                alts = await self.get_alt_response(self.client, ctx, f"{ctx.author.mention} add alts of users\ntype None if not applicable \nFormat \"Alts: handle_1 handle_2 .. \n\"len(handles) <= 2*len(users) must be satisfied", 30, ctx.author)
+                First = False
+            else:
+                alts = await self.get_alt_response(self.client, ctx, f"{ctx.author.mention} add alts of users", 30, ctx.author)
+            if alts[0]:
+                alts = alts[1]
+                if len(alts) < 1:
+                    continue
+                if len(alts) == 1:
+                    if alts[0] == "None":
+                        check = True
+                        alts = []
+                        break
+                else:
+                    alts = alts[1:]
+                    check = True
+                    for alt in alts:
+                        res = await self.api.check_handle(alt)
+                        if not res[0]:
+                            await ctx.send(f"{ctx.author.mention} " + alt + " is not valid codeforces handle, try again")
+                            check = False 
+                            break
```

Comment:
why do you break here, just print the error and wait for new input

Reply:
the break is from checking if multiple handles are wrong we break out after the first invalid handle.
we take new input too